base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseLifecycleManager.java

```diff
@@ -923,6 +934,12 @@ public void onSessionEnded(int sessionID) {
         public void onAuthTokenReceived(String token, int sessionID) {
             BaseLifecycleManager.this.authToken = token;
         }
+
+        @Override
+        public boolean onVehicleTypeReceived(@Nullable VehicleType type, @Nullable String systemSoftwareVersion, @Nullable String systemHardwareVersion) {
```

Comment:
@santanamk, we don't need to know about the manager, because the particular one will be called using delegated instances, hence the reference is redundant as for me